vms/platformvm/txs/executor/proposal_tx_executor.go

```diff
@@ -553,7 +553,7 @@ func (e *ProposalTxExecutor) RewardValidatorTx(tx *txs.RewardValidatorTx) error
 
 		// Reward the delegatee here
 		if delegateeReward > 0 {
-			if vdrStaker.StartTime.After(e.Config.CortinaTime) {
+			if e.Config.IsCortinaActivated(vdrStaker.StartTime) {
```

Comment:
Note that this is a behavior change only when `vdrStaker.StartTime` is equal to `CortinaTime`

Reply:
@StephenButtolph I can't recall why we didn't go for the IsCortinaActivate before. IIRC we chose to stay with the dust rewards if validator.StartTime matched fork activation time.